Firebase/InstanceID/FIRInstanceIDTokenManager.h

```diff
@@ -55,7 +54,7 @@ typedef NS_OPTIONS(NSUInteger, FIRInstanceIDInvalidTokenReason) {
  */
 - (void)fetchNewTokenWithAuthorizedEntity:(NSString *)authorizedEntity
                                     scope:(NSString *)scope
-                                  keyPair:(FIRInstanceIDKeyPair *)keyPair
+                                      IID:(NSString *)instanceID
```

Comment:
maybe instanceID:(NSString *)instanceID to fit better on the style guide.

Reply:
Updated here and at other places